docs/markdown/icons-in-md.md

```diff
@@ -0,0 +1,204 @@
+- Pair text and icons when possible to provide better clarity.
+- Do center-align icons when they’re next to text.
+- Do match your icon color with your text color when pairing them.
+
+**Dont**
+
+- Repurpose icons. Using icons for their intended meaning supports good comprehension.
+- Don't create interactive Icons using TapArea. Use [IconButton](https://gestalt.netlify.app/iconbutton) instead.
+- Don’t baseline-align icons to the text.
+- Don’t use different colors for text and icons.
+
+**Tips for Designing with Icons**
+
+If you decide to include icons in your interface, **do your research first**! Familiarize yourself with **icons used by other competitors** and with icons commonly used **on the platforms**, as those will be most recognizable to your users.
+
+If you must design new icons, strive to make them easy to recognize and memorize by following these guidelines:
+
+- Keep the design **simple and schematic**. Reduce the amount of graphic details by focusing on the basic characteristics of the object rather than creating a highly realistic image in order to speed up recognition. (Intricate details are difficult to distinguish at smaller sizes.)
+- Use the**5-second rule**: if it takes you more than 5 seconds to think of an appropriate icon for something, it is unlikely that an icon can effectively communicate that meaning.
+- [Test the icons for recognizability](https://www.nngroup.com/articles/icon-testing/): ask people what they expect the icons to stand for.
+- **Test the icons for memorability**: ask a repeat set of users if they can remember the icon’s meaning after being told what it represented a couple weeks earlier.
+
+
+## Icon sizing
+
+These are the guidelines for icon sizes (in px):
+12
+Used sparingly in tight spaces. When possible, use a text label next to the icon, as it can be hard to see for visually impaired people.
+14
+Used often following body text content any time an icon is needed.
+16
+Used often any time an icon is needed. Default icon size.
+24
+Used frequently any time an icon is needed.
+32
+Used occasionally, on more dense UI.
+32+
+Should be used sparingly and only in places where the UI is very dense and a larger icon is required.
+
+**Touch targets**
+All touch targets for interactive icons need to be 44px or larger. Engineers can add padding to a touch target to meet the 44px requirement.
+
+
+
+## Icon color
+
+Icons can be created using the following color options. brandPrimary should only be used to represent the Pinterest logo, as it is not accessible. See the [design tokens](https://gestalt.netlify.app/design_tokens#Text-color) for more info.
+
+
+![Screen Shot 2022-05-18 at 1.42.18 PM.png](https://codahosted.io/docs/vlenTJHtyS/blobs/bl-bSZTauMmHr/c77129a1c3ece0e8b0ddca9761f701ab7182dd1b97dd1283c8caf291816e9ead43a0fb6741112ce4c49d582f257cd3ef8a327d2ef60f475febe38fc69d67fefe760e9baa738d7bcb19a8812417ede1ed6a70ff0d3da9667d57d79838360ef9b5117c03ce)
```

Comment:
An image definitely isn't ideal here…but perhaps this page isn't a good candidate for MDX anyway. We could potentially use actual components here?

Reply:
Yeah totally! We'll have to make sure the Code component renders better